paddle/rockcraft.yaml

```diff
@@ -0,0 +1,64 @@
+# Based on https://github.com/kserve/kserve/blob/release-0.11/python/paddle.Dockerfile
+name: paddle-server
+summary: Paddle server for Kserve deployments
+description: "Kserve Paddle server"
+version: "0.11.0"
+license: Apache-2.0
+base: ubuntu@22.04
+platforms:
+    amd64:
+run-user: _daemon_
+services:
+  paddle-server:
+    override: replace
+    summary: "Paddle server service"
+    startup: enabled
+    command: "python3.10 -m paddleserver [ ]"
+    environment:
+      PYTHONPATH: /paddleserver:/kserve
+entrypoint-service: paddle-server
+
+parts:
+  security-team-requirement:
+    plugin: nil
+    override-build: |
+      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
+      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && \
+      dpkg-query --root=${CRAFT_PROJECT_DIR}/../bundles/ubuntu-22.04/rootfs/ -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) \
+      > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
+
+  python:
+    plugin: nil
+    source: https://github.com/kserve/kserve.git
+    source-subdir: python
+    source-tag: v0.11.0
+    build-packages:
+    - python3.10
```

Comment:
Why do we use `build-packages` here and not `stage-packages`?  I'm still digesting, but I think what we are doing here is:
* putting python3.10 into the part for building step only
* during build step, installing packages that end up in the `/usr/local/lib/python3.10/dist-packages`
* priming the `/usr/local/lib/python3.10/dist-packages` **but not the python3.10 we used to construct that dir**
* in the final image, using the final image's native python3.10 which is accessing the above mentioned `/usr/local/lib/python3.10/dist-packages`

is that how this works?

Reply:
yes fresh python3.10 ... I remember rockcraft team recommending that with pmml server to get rid of unnecesery packages from build e.g. poetry.

Reply:
Yes I think so

Reply:
Removed